app/views/admin/configs/show.html.erb

```diff
@@ -134,56 +129,142 @@
                                class: "form-control selectpicker" %>
                 <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:authentication_providers][:description] %></div>
               </div>
+
+                <div class="config-authentication__row">
+                  <div class="config-authentication__row--select">
+                    <%= inline_svg_tag("email.svg", class: "crayons-icon", aria: true, title: "Twitter logo") %>
+                    <div>
+                      <p class="config-authentication__row--title crayons-fields__label">
+                        Email address and password
+                      </p>
+                      <p class="config-authentication__row--subtitle">
+                        Default
+                      </p>
+                    </div>
+                  </div>
+                  <fieldset class="config-authentication-form">
+                    <div class="crayons-field--checkbox">
+                      <%= f.check_box :allow_email_password_registration, checked: SiteConfig.allow_email_password_registration, class: "crayons-checkbox" %>
+                      <div>
+                        <%= admin_config_label :allow_email_password_registration %>
+                        <p class="crayons-field__description">
+                          <%= Constants::SiteConfig::DETAILS[:allow_email_password_registration][:description] %>
+                        </p>
+                      </div>
+                    </div>
+                  </fieldset>
+                </div>
```

Comment:
Headers are quite misaligned with rest of the content below:

![image](https://user-images.githubusercontent.com/108287/95572876-e877c680-0a2a-11eb-83a5-a875495ebf50.png)

I'd personally suggest different HTML structure - it would prevent having misalignments and it would also help with accordion effect later on.

Instead of having two rows (one with icon + header and the other with inputs) I'd actually suggest having two columns - one with icon and the other with header and inputs.

```
+--------+-----------------------+
| +----+ | +-------------------+ |
| |icon| | |Twitter            | |
| +----+ | +-------------------+ |
|        | +-------------------+ |
|        | | Form field...     | |
|        | | Form field...     | |
|        | +-------------------+ |
+--------+-----------------------+
```

Reply:
Thanks for this helpful suggestion. Lemme try this with `grid` and see how far along I get...